design/proposals/identity_support.md

```diff
@@ -0,0 +1,254 @@
+
+   From Identity's documentation, it is not clear whether an Authorization header
+   containing the `client_id` and `client_secret` parameters will be accepted as
+   it is by Okta. If the Authz Header is accepted, then no changes are required.
+   If the Authz Header is not accepted, then we could configure the OIDC client
+   to send the required parameters as part of the request body, which is also
+   accepted by Okta.
+
+### User Interface
+[//]: # "Describe user interface (including command structure, inputs/outputs, etc where relevant)"
+
+The user interface for an AuthnOIDC V2 instance representing Identity will be the
+same as one representing Okta.
+
+## Design
+[//]: # "Add any diagrams, charts and explanations about the design aspect of the solution. Elaborate also about the expected user experience for the feature"
+
+Depending on the answers to the questions listed in the [API Differences](#api-differences-okta-vs-identity)
+section, there may be no need to code changes beyond those made to testing.
+
+## Backwards Compatibility
+[//]: # "How will the design of this solution impact backwards compatibility? Address how you are going to handle backwards compatibility, if necessary"
+
+Depending on the answers to the questions listed in the [API Differences](#api-differences-okta-vs-identity)
+section, there may be no need to code changes beyond testing, and will be backwards
+compatible.
+
+## Affected Components
+[//]: # "List all components that will be affected by your solution"
+[//]: # "[Conjur Open Source/Enterprise, clients, integrations, etc.]"
+[//]: # "and elaborate on the impacts. This list should include all"
+[//]: # "downstream components that will need to be updated to consume"
+[//]: # "new releases as these changes are implemented"
+
+- [Conjur Open Source](https://github.com/cyberark/conjur)
+- [Conjur UI](https://github.com/conjurinc/conjur-ui)
+- [Go-based Conjur CLI](https://github.com/cyberark/conjur-cli-go)
+
+## Work in Parallel
+[//]: # "How can we work in parallel for this task? How this can be done effectively without hindering the work of others who are working on different areas of the task."
+[//]: # "For example, can we introduce minimal automation to run basic sanity tests to protect the work of others?"
+
+Given that the bulk of the work required to validate support for Identity will
+be as dev environment and testing updates, the following work can be done in parallel:
+
+1. Enable Identity development across required projects by including Identity in
+   existing development and CI environments.
+   - Conjur
+   - Conjur UI
+   - Conjur CLI Go
+2. Add Identity to testing strategy across required projects
+   - Conjur. See [test cases summary](#functional-tests).
+   - Conjur UI. See [test cases summary](#functional-tests).
+   - Conjur CLI Go. See [test cases summary](#functional-tests).
+
+## Test Plan
```

Comment:
Is there a need for any tests around having a mix of providers configured? Not sure if the original tests had 2 different Okta providers configured to make sure everything was kept straight between the two, but if not, this might be a good time to add some tests with one Okta & one Identity at least to make sure there is no cross contamination.

Reply:
Address this question [here](https://github.com/cyberark/conjur/pull/2730#issuecomment-1454150130).